src/Tests/Tests/epv_test_validate_languages.php

```diff
@@ -79,6 +79,11 @@ public function validateDirectory(array $files)
 			}
 		}
 
+		if (!empty($langs) && !array_key_exists('en', $langs))
+		{
+			$this->output->addMessage(OutputInterface::FATAL, sprintf("English language pack is missing"));
```

Comment:
should be:
```
$this->output->addMessage(OutputInterface::FATAL, 'English language pack is missing');
```

Reply:
That's just a formality, and double quotes are already used in this file in lines 92 and 100 (numbering with my PR), so what

Reply:
You're calling the `sprintf` function for no reason, plus with double quotes, just adding extra processing that's not needed.

Reply:
I think it also should be a error instead of a fatal

Reply:
@iMattPro: This seems to be correct, but that is easily fixed. (And has - BTW - nothing to do with the problem I've tried to fix)

@paul999: If an extension doesn't have a en language pack (nglish is the fallback language for every situation where a language key could not be found in the chosen language) then phpBB will terminate with a fatal error. This is a good enough reason to make it not an error but a fatal.

Reply:
The sprintf command has been removed.